pkgs/dart_mcp/example/dash_client.dart

```diff
@@ -224,6 +239,27 @@ final class DashClient extends MCPClient with RootsSupport {
     }
   }
 
+  /// Listens for log messages on all [serverConnections] that support logging.
+  void _listenToLogs() {
+    for (var connection in serverConnections) {
+      if (connection.serverCapabilities.logging == null) {
+        continue;
+      }
+
+      connection.setLogLevel(
+        SetLevelRequest(
+          level: verbose ? LoggingLevel.debug : LoggingLevel.warning,
+        ),
+      );
+      connection.onLog.listen((event) {
+        print(
+          'Server Log(${event.level.name}): '
+          '${event.logger != null ? '[${event.logger}] ' : ''} ${event.data}',
```

Comment:
nit: remove the additional space before `${event.data}`. Otherwise, when event.logger == null, the log will have a leading space